.eslintrc.json

```
@@ -1,21 +1,19 @@
 {
-    // lodashは無視するようにした
-    "globals": { "_": true },
-    "root": true,
-    "env": {
-        "es2021": true,
-        "node": true
-    },
-    "extends": [
-        "@nuxtjs/eslint-config-typescript",
-        "plugin:nuxt/recommended"
-    ],
-    "ignorePatterns":[
-        "dist/",
-        "mock/"
-    ],
-    "rules": {
-        "brace-style": ["error", "1tbs", { "allowSingleLine": false }],
-        "curly": ["error", "all"]
-    }
+  // lodashは無視するようにした
+  "globals": { "_": true },
+  "root": true,
+  "env": {
+    "es2021": true,
+    "node": true
+  },
+  "parserOptions": {
+    "parser": "babel-eslint"
+  },
+  "extends": ["plugin:nuxt/recommended"],
```

`@nuxtjs/eslint-config-typescript`の代わりに`@nuxtjs/eslint-config`をインストールしてください。
```suggestion
  "extends": ["plugin:nuxt/recommended", "@nuxtjs"],
```

---

追記しておきます